pyarr/radarr.py

```diff
@@ -0,0 +1,518 @@
+from .base import BaseAPI
+
+
+class RadarrAPI(BaseAPI):
+    """API wrapper for Radarr endpoints.
+
+    Args:
+        RequestAPI (:obj:`str`): provides connection to API endpoint
+    """
+
+    def __init__(self, host_url: str, api_key: str):
+        """Initialise Readarr API
+
+        Args:
+            host_url (str): URL for Readarr
+            api_key (str): API key for Readarr
+        """
+
+        ver_uri = "/v3"
+        super().__init__(host_url, api_key, ver_uri)
+
+    def _construct_movie_json(
```

Comment:
The more I look at the `_construct_*_json` function names the less appealing they look.

I think it might be best to remove `_construct` from the name.

Reply:
agreed, this isnt needed